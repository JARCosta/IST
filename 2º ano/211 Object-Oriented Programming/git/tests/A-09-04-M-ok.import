PARTNER|AAAAS1|Toshiba|Tokyo, Japan
PARTNER|BBBBW2|Pedraria Fonseca|Oeiras, Portugal
PARTNER|CCCCCCCCP1|Lages do Ch√£o|Lisboa, Portugal
PARTNER|DDDDDDP3|ObBrian Rocks|Koln, Germany
PARTNER|EEEEEER2|Jorge Figueiredo|Lisboa, Portugal
PARTNER|EFFFFF4|Filomena Figueiredo|Lisboa, Portugal
