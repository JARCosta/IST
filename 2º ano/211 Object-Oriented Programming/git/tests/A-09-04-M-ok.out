Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Parceiros
7 - Gestão de Transacções
8 - Menu de Consultas
9 - Ver Saldo Global
0 - Sair
Escolha uma opção: Gestão de Produtos
1 - Visualizar todos os produtos
2 - Visualizar todos os lotes
3 - Visualizar lotes fornecidos por parceiro
4 - Visualizar lotes de um dado produto
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Parceiros
7 - Gestão de Transacções
8 - Menu de Consultas
9 - Ver Saldo Global
0 - Sair
Escolha uma opção: Gestão de Parceiros
1 - Mostrar parceiro
2 - Mostrar parceiros
3 - Registar parceiro
4 - Activa/desactivar notificações de um produto
5 - Mostrar compras com parceiro
6 - Mostrar vendas (e desagregações) com parceiro
0 - Sair
Escolha uma opção: Identificador do parceiro: Nome do parceiro: Endereço do parceiro: Gestão de Parceiros
1 - Mostrar parceiro
2 - Mostrar parceiros
3 - Registar parceiro
4 - Activa/desactivar notificações de um produto
5 - Mostrar compras com parceiro
6 - Mostrar vendas (e desagregações) com parceiro
0 - Sair
Escolha uma opção: Identificador do parceiro: Nome do parceiro: Endereço do parceiro: Registar parceiro: Operação inválida: O parceiro 'YYYYYYYY' já existe.
Gestão de Parceiros
1 - Mostrar parceiro
2 - Mostrar parceiros
3 - Registar parceiro
4 - Activa/desactivar notificações de um produto
5 - Mostrar compras com parceiro
6 - Mostrar vendas (e desagregações) com parceiro
0 - Sair
Escolha uma opção: Identificador do parceiro: Nome do parceiro: Endereço do parceiro: Registar parceiro: Operação inválida: O parceiro 'CCCCCCCCP1' já existe.
Gestão de Parceiros
1 - Mostrar parceiro
2 - Mostrar parceiros
3 - Registar parceiro
4 - Activa/desactivar notificações de um produto
5 - Mostrar compras com parceiro
6 - Mostrar vendas (e desagregações) com parceiro
0 - Sair
Escolha uma opção: AAAAS1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
BBBBW2|Pedraria Fonseca|Oeiras, Portugal|NORMAL|0|0|0|0
CCCCCCCCP1|Lages do Chão|Lisboa, Portugal|NORMAL|0|0|0|0
DDDDDDP3|ObBrian Rocks|Koln, Germany|NORMAL|0|0|0|0
EEEEEER2|Jorge Figueiredo|Lisboa, Portugal|NORMAL|0|0|0|0
EFFFFF4|Filomena Figueiredo|Lisboa, Portugal|NORMAL|0|0|0|0
YYYYYYYY|wer|rwer|NORMAL|0|0|0|0
Gestão de Parceiros
1 - Mostrar parceiro
2 - Mostrar parceiros
3 - Registar parceiro
4 - Activa/desactivar notificações de um produto
5 - Mostrar compras com parceiro
6 - Mostrar vendas (e desagregações) com parceiro
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Parceiros
7 - Gestão de Transacções
8 - Menu de Consultas
9 - Ver Saldo Global
0 - Sair
Escolha uma opção: 